### DISCUSSION TITLE
Taskbar Window doesn't appear on LG C2 OLED TV IN HDR MODE Windows 11

### DISCUSSION THREAD

**Discussion by TheExecutionerGH** [zhongyang219/TrafficMonitor#1608] [Q&A]
I have this issue regarding Taskbar Window not appearing on LG C2 OLED TV IN HDR MODE Windows 11. The taskbar window appears only in SDR mode; when I switch to HDR mode, the only visible thing is Main Window.

Is there a way to make a visible taskbar window in HDR mode?

**Comment by hhht110** [comment]
对 我也遇到了 我刚刚换了个电脑然后装了win10.发现开启hdr以后色彩准确的都很不错 桌面啥的很清晰，结果偏偏这个软件任务栏不显示了。。。你2023年就反馈了。。都没解决？
